debezium-connector-cassandra/src/main/java/io/debezium/connector/cassandra/CommitLogProcessor.java

```diff
@@ -115,11 +121,14 @@ void processCommitLog(File file) throws IOException {
                 }
                 LOGGER.info("Successfully processed commit log {}", file.getName());
             }
-            catch (IOException e) {
+            catch (Exception e) {
                 if (!latestOnly) {
                     queue.enqueue(new EOFEvent(file, false));
                 }
                 LOGGER.warn("Error occurred while processing commit log " + file.getName(), e);
```

Comment:
This should be warning only when exception is not rethrown, otherwise it should be error.

Reply:
> This should be warning only when exception is not rethrown, otherwise it should be error.

I see, updated accordingly

Reply:
I think was not fully clear, my intent was something like

```
if (...) {
    LOGGER.error
    throw e;
}
LOGGER.warn
```

Also could you please rebase upon the lastest master? There was a change in the `debezium` repo breaking builds for this one.

Reply:
> I think was not fully clear, my intent was something like
> 
> ```
> if (...) {
>     LOGGER.error
>     throw e;
> }
> LOGGER.warn
> ```
> 
> Also could you please rebase upon the lastest master? There was a change in the `debezium` repo breaking builds for this one.

Updated accordingly!